tembo-cli/examples/merge/overlay.toml

```diff
@@ -1,4 +1,4 @@
-[merge]
-environment = "prod"
+[defaults]
```

Comment:
@joshuajerin Can you keep this as `merge` instead of `defaulta`. This was a PR comment Steven provided so I made those changes in [this](https://github.com/tembo-io/tembo/pull/487) PR